src/Compilers/CSharp/csc/project.json

```diff
@@ -4,7 +4,9 @@
     "net45": { }
   },
   "runtimes": {
+    "win7": { },
     "win7-anycpu": { },
```

Comment:
Should we be removing these, or are you leaving them for compat with VS2015 RTM?